sanic/server/websockets/impl.py

```diff
@@ -21,7 +21,7 @@
 
 
 try:  # websockets < 11.0
-    from websockets.connection import Event, State
+    from websockets.connection import Event, State  # type: ignore
```

Comment:
Why's this needed now and it wasn't before?

Reply:
Because it is not importable from that location (which is precisely why we have the try/except). mypy upgrade now complains and is not smart enough to capture the try/except logic.